app/src/main/java/org/p2p/wallet/home/MainFragment.kt

```diff
@@ -128,6 +132,17 @@ class MainFragment : BaseFragment(R.layout.fragment_main), MainTabsSwitcher, Cen
     }
 
     override fun navigate(clickedTab: ScreenTab) {
+        if (clickedTab == ScreenTab.FEEDBACK_SCREEN) {
+            IntercomService.showMessenger()
+            analyticsInteractor.logScreenOpenEvent(ScreenNames.Main.MAIN_FEEDBACK)
+            with(binding.bottomNavigation) {
+                post { // not working reselection on last item without post
+                    setChecked(lastSelectedItemId)
+                }
+            }
+            return
```

Comment:
what if you close intercom, does the bottom navbar refresh or it's still be checked on feedback?

Reply:
refreshes, feedback button is not checkable